### DISCUSSION TITLE
Can Sandboxie Classic and Sandboxie Plus portable live together?

### DISCUSSION THREAD

**Discussion by science2002** [sandboxie-plus/Sandboxie#3884] [Documents and notes]
Sandboxie is a great piece of software. Thank you so much to keep it alive.
I have this scenario, which seems not so uncommon.

- I have installed (normal installation, with just the default settings) Sandboxie Classic in Pc1 (i.e. C:\ drive). I use it mainly with the browser
- I have also Sandboxie Plus Portable in an exernal HDD with some software within it (Sandbox folder is in the same portable drive, alongside the Sandboxie Plus. There are several containers: one for each  application I installed within SB Plus portable.  I use this version of SB between Pcs.

Now I tried to run one app in it  (from Drive M:) in Pc1 and it messed up the functioning of Sandboxie Classic: I had problems with the browser: it did not see all the critical web files (bookmarks etc.) saved in C:\Sandbox, which are vital for me. I believe that SB Plus, despite being portable, wrote something in the Pc1 registry.

Question: Is it possible to use a SB Plus portable, in a Pc that has already a normal installation of Sandboxie Classic? What is the best practice in this case, to avoid conflicts, and keep them separate?
Thanks
PS: I asked this question to forum.xanasoft.com, but I was indirectly sent for the answer here to the developer.

**Comment by offhub** [comment]
Sandboxie does not support using different installation modes at the same time. You have to use Sandboxie in either installed or portable mode. There are no tools in the Sandboxie Classic interface to facilitate this transition. So it would be better to use Sandboxie Plus for both cases.

To make Sandboxie Plus fully portable, you must first remove the driver and service of the installed version from the system. You can do this with `Sandboxie Plus > Sandbox > Maintenance > Uninstall All`. For the portable version, this process is done by asking for your permission when closing the interface.

If you do not want to deal with the above processes, you can create and use a configuration that includes both installed and portable settings.

1. Close all sandboxed programs.
2. Backup the `C:\Windows\Sandboxie.ini` file.
3. Replace it with the configuration you want to use.
4. Reload the configuration using `Options > Reload configuration`.
5. Repeat the above steps if you want to use a different configuration.

**Reply by science2002** [reply]
Thanks @offhub for your answer. 
As you can immagine, my aim was not (and is not) to run two sandboxie per se, but it was to run the sandboxed software from the  SB+Portable, in a Pc that already contains a SBClassic.

You mention the problem of Sandboxie.ini. I will look at it better. But I notice that SB Classic - being the app fully installed - does show it as expected in Windows folder. Yet the SB+Portable is present within its own folder (in drive M: in my case). 

I could be wrong, but I supspected in my limited test, that the mess up of SB+port. with the SBClassic was due to some conflicts in the registry. I believed in fact - but you are warning me against this my assumption - that being the driver and the service already present in Pc1 - from SBClassic - the SB+ could work with them, while the  two Sandboxie.ini placed in two separate folders could avoid confusion. However from your words this may not be the case.

How to try to implement your point 3. wil not be easy for me. I start studying here:
https://sandboxie-plus.github.io/sandboxie-docs/
Please let me know if there are other good sources to catch up with  on this matter.
Thanks again.
